radio/src/targets/taranis/diskio.cpp

```diff
@@ -978,7 +978,7 @@ void sdPoll10ms()
 
 FATFS g_FATFS_Obj;
 #if defined(LOG_TELEMETRY)
-FIL g_telemetryFile = {0};
+FIL g_telemetryFile;

```

Comment:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119

As I understand it, the "proper" initializer in C++ should be `= {}` (although this is fixed in g++5).  And indeed that silences the warning.  I'll change both instances.